ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -370,15 +370,8 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{ na
     }
 
     private async updateApp(app: appModels.Application) {
-        try {
-            await services.applications.updateSpec(app.metadata.name, app.spec);
-            this.refreshRequested.next({});
-        } catch (e) {
-            this.appContext.apis.notifications.show({
```

Comment:
did you want to delete this catch?

Reply:
Yes, the end user of this function has catch and error handler: panel which edit parameters/general app fields shows error message if save fails and don't switch back to view mode if server side validation fails.